chirp/common/printing.py

```
@@ -0,0 +1,30 @@
+import contextlib
+
+
+class CustomPrint(object):
+    """
+    This class defines a callable object that works similarly to the print()
+    function. However, it provides a context manager that allows you to specify
+    a different write function that can be used to send messages to somewhere
+    other than standard output (for example, a web interface).
+
+    """
+
+    def __init__(self):
+        self.write = self.default_write
+
+    def __call__(self, message=None, **kwargs):
+        self.write(message, **kwargs)
+
+    def default_write(self, message=None, **kwargs):
```

is there a legitimate case where message would be `None`? Or does this satisfy some tornado interface? I think if the caller were to pass in a `None` message then it should raise an error because that would probably be a programming mistake.